admin/index.php

```diff
@@ -90,7 +90,14 @@ function bounceOut(){ header("Location: http://davesmapper.com/admin"); exit; }
 	}
 	if ($_POST['action'] == "add_carto") {
 		mysql_query("INSERT INTO artists (name, url_slug, initials, icon, url, bio, email, password)
-								 VALUES ('".mysql_real_escape_string($_POST['aname'])."', '".mysql_real_escape_string($_POST['aurlslug'])."', '".mysql_real_escape_string($_POST['ainit'])."', '".mysql_real_escape_string($_POST['aicon'])."', '".mysql_real_escape_string($_POST['alink'])."', '".mysql_real_escape_string($_POST['abio'])."', '".mysql_real_escape_string($_POST['aemail'])."', '".sha1($_POST['aemail']."bacon"."c00lt1l3sbr0")."')");
+								 VALUES ('".mysql_real_escape_string($_POST['aname'])."',
+								 				 '".mysql_real_escape_string($_POST['aurlslug'])."',
+								 				 '".mysql_real_escape_string($_POST['ainit'])."',
+								 				 '".mysql_real_escape_string($_POST['aicon'])."',
+								 				 '".mysql_real_escape_string($_POST['alink'])."',
+								 				 '".mysql_real_escape_string($_POST['abio'])."',
+								 				 '".mysql_real_escape_string($_POST['aemail'])."',
+								 				 '".sha1($_POST['aemail']."bacon".$_POST['apass'])."')");
```

Comment:
Just realized the over-indentation was because indentation was done with tabs instead of spaces. Fixed that in my editor :)